webdriver-spec.html

```diff
@@ -3015,8 +3015,8 @@
  the operating system notion of a “window”
  or the DOM <a><code>Window</code></a> object.
 
-<p>The <a>top-level browsing context</a> is said
- to be <dfn>no longer open</dfn> if it has been <a>discarded</a>.
+<p>A <a>browsing context</a> is said to be <dfn>no longer open</dfn>
+ if it has been <a>discarded</a>.
```

Comment:
I wonder if we for browsing contexts (not top-level) should just use the HTML definition _discarded_. I added _no longer open_ for top-levels to sort of indicate that the OS window had been closed, but for _browsing context_ checks in commands, we typically mean “is the DOM the user chose still available? ”, or “is the `<iframe>` still there?”